assets/src/js/plugins/give-api/notice.js

```diff
@@ -90,14 +90,22 @@ export default {
 		 * Note: use only in WP Backend
 		 *
 		 * @since 2.5.0
+		 * @since 2.8.0 Localization updated to support changes in WordPress 5.5.
+		 *
 		 * @param {string} notice Notice description.
 		 * @param {string} type   Notice type.
 		 * @param {object} args   Notice type.
 		 *
 		 * @return {string} Notice HTML.
 		 */
 		getAdminNoticeHTML: function( notice, type = 'info', args = { dismissible: true } ) {
-			const btnText = commonL10n.dismiss || '';
+			/**
+			 * WordPress 5.5 removed the localized `commonL10n` in favor of translation in JavaScript.
+			 */
+			const btnText = ( 'undefined' !== typeof commonL10n ) ?
```

Comment:
@kjohnson This text comes from PHP which is already translated, do we have to retranslate it in js?

![image](https://user-images.githubusercontent.com/1784821/91063378-8b2ae000-e64b-11ea-9074-27d61b7668ff.png)